src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -1,49 +1,67 @@
 package nl.tudelft.ewi.se.ciselab.testgenie.toolwindow
 
-import com.intellij.openapi.ui.Messages
-
-import java.awt.Dimension
-import java.awt.GridBagLayout
-import java.awt.event.ActionEvent
-import javax.swing.JButton
 import com.intellij.openapi.ui.ComboBox
+import com.intellij.openapi.ui.Messages
 import com.intellij.ui.components.JBLabel
 import com.intellij.ui.components.JBScrollPane
 import com.intellij.util.ui.FormBuilder
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+import org.jdesktop.swingx.JXTitledSeparator
 import java.awt.*
+import java.awt.event.ActionEvent
 import javax.swing.*
 
 /**
  * This class stores the UI of the TestGenie tool window.
  */
 class TestGenieToolWindow {
```

Comment:
Maybe also change the names of the classes since this doesn't involve the entire tool window but only the parameters tab.

Reply:
This comment was addressed by Jegor and you in the code. I think that everything should be fine now.